tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -220,36 +326,45 @@ def test_OnDiskDataset_TVTSet_ItemSet_node_pair_label():
                 train_set:
                   - type: null
                     data:
-                      - format: numpy
+                      - name: src
```

Comment:
We need to make sure that user knows our keywords, currently the only mis-match between the reserved name and mini-batch variables is src, dst with node_pairs. Is this the best option?

Reply:
create a ticket for this issue: https://github.com/dmlc/dgl/issues/6259.